plugin/trino-hive/src/test/java/io/trino/plugin/hive/AbstractTestHive.java

```diff
@@ -799,7 +799,8 @@ protected final void setup(String host, int port, String databaseName, String ti
                 executor,
                 new Duration(1, MINUTES),
                 Optional.of(new Duration(15, SECONDS)),
-                10000);
+                10000,
+                true);
```

Comment:
ok

Reply:
@s2lomon 
if this tests's purpose is  to test things related to partitions being cached or not, then i agree, the config should be set explicitly.
however, it's not the goal here. you supplied the config only because the compiler ordered you to (missing parameter), not that because you really wanted to enable or disable partition caching.
`new ..Config().is...CacheaPartitions()` conveys this intent, and neither `true` nor `false`  conveys that intent

Reply:
switch to `new ..Config().is...CacheaPartitions()` please

Reply:
Omg you are right, for some reason I've thought that this comment was about the default test. Good point and thanks for being patient :)